src/components/book-preview.ts

```diff
@@ -1,38 +1,53 @@
 import { Component, Input } from '@angular/core';
 
 import { Book } from '../models';
+import { MD_CARD_DIRECTIVES } from '@angular2-material/card';
+import { MD_LIST_DIRECTIVES } from '@angular2-material/list';
 
 
 export type BookInput = Book;
 
 @Component({
   selector: 'book-preview',
+  directives: [
+    ...MD_CARD_DIRECTIVES,
```

Comment:
You don't have to spread out the arrays here. `directives` can be a deeply nested array. Change to:

```ts
directives : [
  MD_CARD_DIRECTIVES,
  MD_LIST_DIRECTIVES
]
```